### PR TITLE
spell checker

### PR DESCRIPTION
**Pull request by 0xMuluh**
- minor spelling errors in comments and some docs

### CODE REVIEW

File: R/getPrevalence.R [hunk 1]
```diff
@@ -109,7 +109,7 @@
 #'                                       sort = TRUE)
 #' head(prevalence.frequency)
 #'
-#' # Get prevalence estimates for phylums
+#' # Get prevalence estimates for phylum
```
Comment (antagomir) [inline R/getPrevalence.R]: The plural is "phyla"

File: R/getPrevalence.R [hunk 2]
```diff
@@ -228,7 +228,7 @@ setMethod("getPrevalence", signature = c(x = "ANY"), function(
 .agg_for_prevalence <- function(
         x, rank, relabel = FALSE, make.unique = TRUE, na.rm = FALSE,
         agg.na.rm = TRUE, ...){
-    # Check na.rm. It is not used in this function, it is only catched so that
+    # Check na.rm. It is not used in this function, it is only cached so that
```
Comment (antagomir) [inline R/getPrevalence.R]: are you sure that "cached" is what is meant here - can you double check?

**Review by antagomir — COMMENTED** [review]
some small comments

File: R/getPrevalence.R (hunk 1 above)
Comment (0xMuluh) [reply R/getPrevalence.R]: ACK

File: R/getPrevalence.R (hunk 2 above)
Comment (0xMuluh) [reply R/getPrevalence.R]: Right, the word should be caught. Thanks

**Pull request merged**